packages/smooth_ui_library/lib/widgets/smooth_expandable_card.dart

```diff
@@ -26,8 +26,8 @@ class SmoothExpandableCard extends StatelessWidget {
             duration: const Duration(milliseconds: 160),
             firstCurve: Curves.easeInOutBack,
             secondCurve: Curves.easeInOutBack,
-            firstChild: _buildCollapsedWidget(singleBooleanModel),
-            secondChild: _buildExpandedWidget(singleBooleanModel),
+            firstChild: _buildCollapsedWidget(singleBooleanModel, context),
```

Comment:
For more clarity I would pass a `ThemeData` (=`Theme.of(context)`) instead of a `BuildContext`.